src/views/v2/month/calendar-body/day.php

```diff
@@ -88,15 +102,47 @@ class="tribe-events-calendar-month__day-date-daynum"
 		<?php if ( ! empty( $day['featured_events'] ) ): ?>
 			<em
 				class="tribe-events-calendar-month__mobile-events-icon tribe-events-calendar-month__mobile-events-icon--featured"
-				aria-label="<?php esc_attr_e( 'Has featured events', 'the-events-calendar' ); ?>"
-				title="<?php esc_attr_e( 'Has featured events', 'the-events-calendar' ); ?>"
+				aria-label="<?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 55103c69c99d46626ec0041311284b14</sub>